great_expectations/data_context/store/html_site_store.py

```diff
@@ -107,9 +109,9 @@ def __init__(self, store_backend=None, runtime_environment=None):
         store_class = load_class(store_backend_class_name, store_backend_module_name)
 
         # Store Class was loaded successfully; verify that it is of a correct subclass.
-        if not issubclass(store_class, TupleStoreBackend):
+        if not issubclass(store_class, (TupleStoreBackend, GeCloudStoreBackend)):
```

Comment:
Since we created a new JsonSiteStore, we can revert all of the changes in this module, no?